### ISSUE TITLE
django.core.exceptions.ImproperlyConfigured

### ISSUE THREAD

**Issue by alexsonnay** [ui/django-post_office#244]
`Requested setting POST_OFFICE, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.`

Any idea ?

**Comment by fenilgridle** [comment]
Yup, this error occurs when the django settings cannot be found in the runtime environment. Could you elaborate on how you ended up here ?
